src/core/types/elasticsearch/search.ts

```diff
@@ -536,6 +544,7 @@ export type AggregateOf<
           {
             doc_count: number;
             key: string | number;
+            key_as_string?: string;
```

Comment:
What's the intention of the `key_as_string` optional type? Is it a workaround for the mixed `key: string | number` type declaration?